monitoring/count-items-cronjob/alerts.yaml

```diff
@@ -0,0 +1,66 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: count_items_cronjob
+    type: constant
+    value: artesca-data-ops-count-items
+  - name: count_items_job_duration_threshold
+    type: config
+    value: 3600
+
+groups:
+  - name: count-items-cronjob/alerts.rules
+    rules:
+      - alert: CountItemsJobTakingTooLong
+        Expr: |
+          (
+            time() - 
+            label_replace(
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${count_items_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${count_items_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.+)")
+          ) > ${count_items_job_duration_threshold}
+        Labels:
+          severity: critical
```

Comment:
Is this alert level really critical ? It doesn't prevent any service to work, only delays the computation of the metrics. Alert level might need to be confirmed with Florent.